cmd/machine-config-operator/bootstrap.go

```diff
@@ -66,6 +67,8 @@ func init() {
 	bootstrapCmd.MarkFlagRequired("etcd-image")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.setupEtcdEnvImage, "setup-etcd-env-image", "", "Image for Setup etcd Environment.")
 	bootstrapCmd.MarkFlagRequired("setup-etcd-env-image")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdQuorumGuardImage, "etcd-quorum-guard-image", "", "Image for etcd Quorum Guard.")
```

Comment:
this needs to have a default as per #633 https://github.com/openshift/machine-config-operator/pull/633#discussion-diff-274931807R74

Reply:
otherwise, there won't be any substitution in the yaml and the deployment can't kick off

Reply:
OK

Reply:
OK, in.

Reply:
uhm, but now, the flag is gonna be empty right? and it will fail the check as required below